orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/scalingpolicy/UpsertScalingPolicyTask.groovy

```diff
@@ -16,38 +16,48 @@
 
 package com.netflix.spinnaker.orca.clouddriver.tasks.scalingpolicy
 
+import com.netflix.spinnaker.orca.api.operations.OperationsContext
+import com.netflix.spinnaker.orca.api.operations.OperationsInput
+import com.netflix.spinnaker.orca.api.operations.OperationsRunner
 import com.netflix.spinnaker.orca.api.pipeline.models.StageExecution
 
 import java.util.concurrent.TimeUnit
 import com.netflix.spinnaker.orca.api.pipeline.models.ExecutionStatus
 import com.netflix.spinnaker.orca.api.pipeline.RetryableTask
 import com.netflix.spinnaker.orca.api.pipeline.TaskResult
-import com.netflix.spinnaker.orca.clouddriver.KatoService
 import com.netflix.spinnaker.orca.clouddriver.tasks.AbstractCloudProviderAwareTask
-import org.springframework.beans.factory.annotation.Autowired
 import org.springframework.stereotype.Component
 import groovy.util.logging.Slf4j
 
 @Component
 @Slf4j
 class UpsertScalingPolicyTask extends AbstractCloudProviderAwareTask implements RetryableTask {
 
-  @Autowired
-  KatoService kato
-
   long backoffPeriod = TimeUnit.SECONDS.toMillis(5)
   long timeout = TimeUnit.SECONDS.toMillis(100)
 
```

Comment:
can we name the variable as `operationsRunner` ?

Reply:
Oh yep, missed this during refactors.